app/scripts/chart-factory/google-chart-factory.js

```diff
@@ -7,70 +6,52 @@
  * # Extending Gobal Chart Factory for Google Chart Model
  *
  */
-
-angular.module('apacheZeppelinGsocApp').factory('GoogleChartFactory', function(ChartFactory) {
-
-
-  var ChartList = {'Bar':'BarChart','Line':'LineChart'};
+angular.module('apacheZeppelinGsocApp').factory('GoogleChartFactory', function(
+  ChartFactory) {
+  var ChartList = {
+    'Bar': 'BarChart',
+    'Line': 'LineChart'
+  };
   //googleChart model (sample chart data model)
   //TO-DO Sample Data will remove after model set.
   var GoogelChartChartModel = {
     type: 'BarChart',
-    cssStyle : 'height:400px; width:600px;',
-    data : {
-    'cols': [{
-    id: 'pizza',
-    label: 'Pizza',
-    type: 'string'
-}, {
-    id: 'populartiy',
-    label: 'Populartiy',
-    type: 'number'
-}
-
-    ],
-    'rows': [{
-c: [{
-    v: 'Pepperoni'
-}, {
-    v: 14
-}]
-    }, {
-c: [{
-    v: 'Mushroom'
-}, {
-    v: 6
-}]
-    }, {
-c: [{
-    v: 'Hawaiian'
-}, {
-    v: 5
-}]
-    }, {
-c: [{
-    v: 'Sausage'
-}, {
-    v: 10
-}]
-    }]
-},
-  options : {
-    'isStacked': 'true',
-    'fill': 20,
-    'height': 300,
-    'displayExactValues': true,
-    'vAxis': {
-    'gridlines': {
-    'count': 6
-}
+    cssStyle: 'height:400px; width:500px;',
+    data: {
```

Comment:
Those lines were there for model building.  When It is done It is shown in comment